recipes/pyvisa-py/meta.yaml

```diff
@@ -0,0 +1,74 @@
+package:
+  name: pyvisa-py
+  version: "0.2"
```

Comment:
Please check the example to improve this recipe maintainability. We prefer to have at least the version as a `jinja2` variable for easier updates.